README.md

```diff
@@ -23,28 +23,41 @@ which tuna or marlin was brought on board as in [this demo](https://www.youtube.
 
 
 --------
+--------
 
 ## Running the Importer
 
+To get a simple overview of importer and how to use it, try running this from a shell:
+
 ```sh
-usage: importer.py [-h] [-v] [-r REGEX] [-c] [-s STORAGE] [-f FOLDER]
-                   [-i HOST] [-p PORT] [-m HOOK_MODULE]
+python importer.py --help
+```
+
+Which will output:
+
+```man
+usage: importer.py [-h] [-r REGEX] [-c] [-s STORAGE] [-f FOLDER] [-i HOST]
+                   [-p PORT] [-m HOOK_MODULE] [-d HOOK_DATA_JSON] [-v] [-q]
 
 optional arguments:
   -h, --help            show this help message and exit
-  -v, --verbose         log more info
   -r REGEX, --regex REGEX
-                        regex to find camera name and timestamp
+                        regex to find camera name
   -c, --csv             dump csv log file
   -s STORAGE, --storage STORAGE
                         location of the local storage db
   -f FOLDER, --folder FOLDER
-                        folder (directory) to process
-  -i HOST, --host HOST  the segmenter's IP address
-  -p PORT, --port PORT  the segmenter's port number
+                        folder to process
+  -i HOST, --host HOST  the segmenter ip
```

Comment:
@jhallard is this limited to IP address or can it also be a host name on the local network network?